ext/compressors/iaa/iaacodec/src/IAACompressionCodecDeflate.cpp

```diff
@@ -58,13 +59,25 @@ DeflateJobHWPool::~DeflateJobHWPool()
 HardwareCodecDeflate::HardwareCodecDeflate(WT_COMPRESSOR *compressor, WT_SESSION *session)
 {
     hwEnabled = DeflateJobHWPool::instance(compressor, session).jobPoolReady();
+    pageSize = (int)getpagesize();
```

Comment:
This cast is not necessary - `getpagesize()` already returns the correct type.

Reply:
Hi @Meilianggang - I've pushed a fix for this and made sure it builds locally. If the testing passes I'll merge it.